rosbag2_storage/include/rosbag2_storage/storage_interfaces/read_only_interface.hpp

```diff
@@ -28,7 +27,7 @@ namespace storage_interfaces
 {
 
 class ROSBAG2_STORAGE_PUBLIC ReadOnlyInterface
-  : public BaseInfoInterface, public BaseIOInterface, public BaseReadInterface
+  : public BaseIOInterface, public BaseReadInterface
 {
 public:
   virtual ~ReadOnlyInterface() = default;
```

Comment:
```suggestion
   virtual ~ReadOnlyInterface() = default;
  
```

Reply:
The same change should be done in `read_write_interface.hpp` and `base_io_interface.hpp`

Reply:
I hopefully found all occurences

Reply:
The change above is outdated. The interface is back in, so the only line changing in that file is the addition of a newline.